service/src/test/java/org/ehrbase/service/ValidationServiceTest.java

```diff
@@ -39,21 +42,26 @@
     public CacheRule cacheRule = new CacheRule();
 
     @Test
-    public void testCheckComposition() throws Exception {
+    public void testCheckComposition(){
+        try {
+            KnowledgeCacheService knowledgeCacheService = KnowledgeCacheServiceTest.buildKnowledgeCache(testFolder, cacheRule);
+            knowledgeCacheService.addOperationalTemplate(IOUtils.toByteArray(Files.newInputStream(Paths.get("./src/test/resources/knowledge/RIPPLE-ConformanceTest.opt"))));
```

Comment:
Please Use OperationalTemplateTestData.RIPPLE_CONFORMANCE_TEST.getStream()

Reply:
fixed with the new commit